bigquery/google/cloud/bigquery/dbapi/cursor.py

```diff
@@ -17,6 +17,7 @@
 import collections
 import uuid
 
+import google.cloud.exceptions
```

Comment:
@jonparrott We've typically kept our imports **after** 3rd-party, but do what you think is appropriate